awx/ui/src/components/Schedule/shared/ScheduleForm.js

```diff
@@ -124,17 +142,17 @@ function ScheduleFormFields({ hasDaysToKeepField, zoneOptions }) {
       <FormGroup
         name="timezone"
         fieldId="schedule-timezone"
-        helperTextInvalid={timezoneMeta.error}
+        helperTextInvalid={timezoneMeta.error || timezoneMessage}
         isRequired
-        validated={
-          !timezoneMeta.touched || !timezoneMeta.error ? 'default' : 'error'
-        }
+        validated={timezoneValidatedStatus}
         label={t`Local time zone`}
+        helperText={timezoneMessage}
       >
         <AnsibleSelect
           id="schedule-timezone"
           data={zoneOptions}
           {...timezone}
+          onChange={warnLinkedTZ}
```

Comment:
Best practice here is to name the function prop you are passing to the onChange event handler as `handleOnChange` rather than something as specific as `warnLinkedTZ`. In actuality, you're doing more than just warning the user that a user has selected a linked time zone (which depends on their selection) - there's a series of validations that occur or there may be other bits of logic that need to slot into place.